install/bash-preexec.sh

```
@@ -0,0 +1,353 @@
+
+    if  __bp_in_prompt_command "${BASH_COMMAND:-}"; then
+        # If we're executing something inside our prompt_command then we don't
+        # want to call preexec. Bash prior to 3.1 can't detect this at all :/
+        __bp_preexec_interactive_mode=""
+        return
+    fi
+
+    local this_command
+    this_command=$(
+        export LC_ALL=C
+        HISTTIMEFORMAT= builtin history 1 | sed '1 s/^ *[0-9][0-9]*[* ] //'
```

Shellcheck (0.10.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)